tap.js

```diff
@@ -37,8 +59,9 @@
             this.el.addEventListener('touchend', this, false);
             this.el.addEventListener('touchcancel', this, false);
 
-        } else if (e.type === 'mousedown') {
+        } else if (e.type === 'mousedown' && this.whichbutton(e) === 1) {
```

Comment:
many browsers dont support "buttons" but it is on the standards track so i put it first